### PR TITLE
Fix EOS get_arp_table for static ARP records

### PR DESCRIPTION
**Pull request by praniq**
For static ARP entries EOS eAPI gives records without "age" key. Therefore NAPALM throws an exception in get_arp_table during:
`age = float(neighbor.get('age'))`

Fixing this bug.

### CODE REVIEW

File: napalm/eos/eos.py [hunk 1]
```diff
@@ -966,7 +966,7 @@ def get_arp_table(self, vrf=""):
             interface = str(neighbor.get("interface"))
             mac_raw = neighbor.get("hwAddress")
             ip = str(neighbor.get("address"))
-            age = float(neighbor.get("age"))
+            age = float(neighbor.get("age", 0.0))
```
Comment (bewing) [inline napalm/eos/eos.py:969]: Should we use `-1.0` to mark static entries?  I notice nxos uses this approach, but IOS doesn't.  Perhaps worth standardizing on?

Comment (praniq) [reply napalm/eos/eos.py:969]: I would vote for -1.0 for all OSes, as looks like "-1" is a standard value for N/A in Napalm.

Comment (mirceaulinic) [reply napalm/eos/eos.py:969]: Agreed. @praniq - please feel free to update this branch, then I can merge it.

Comment (praniq) [reply napalm/eos/eos.py]: Updated EOS, IOS and IOS XR. Please have a look. Thanks

**Review by mirceaulinic — APPROVED** [review]
LGTM. Thanks @praniq

**Pull request merged**